csrc/scheduler/transpose.cpp

```diff
@@ -235,7 +235,7 @@ class DomainMap : public pointwise_utils::DomainMap {
     // transformed. So the mapping here would require a new compute at map to be
     // constructed from the updated fusion. We'll revisit this once our id graph
     // refactor is done.
-    auto mapped_id = getMappedRootDimIn(tv, root_dim);
+    auto mapped_id = getMappedAllocDimIn(tv, root_dim);
```

Comment:
So originally, you were looking for inner domain of group 2 in the global reference's root domain, but now you are looking for it in the allocation domain. For the case where the allocation domain is not a recorder of rFactor, neither will work. But for now, I think it should be fine.

Reply:
Thanks for pointing this out.

I like how fancy our allocation domain support is, yet scheduler are like... thank you but no. :laughing: